modules/custom/mentions/src/Plugin/Filter/MentionsFilter.php

```diff
@@ -275,7 +275,7 @@ public function filterMentions(string $text): string {
             '#render_value' => $output['value'],
             '#render_plain' => $output['render_plain'] ?? FALSE,
           ];
-          $mentions = $this->renderer->renderPlain($build);
+          $mentions = $this->renderer->renderInIsolation($build);
```

Comment:
Considering this lands in our 13 alpha, it's not an issue but thought I'd share anyway.
What we could've done is become forward compatible, this means we're compatible with both 10.2 and 10.3

Using

```
/** @var \Drupal\Core\Render\RendererInterface $renderer */
$renderer = \Drupal::service('renderer');
$output = DeprecationHelper::backwardsCompatibleCall(
  currentVersion: \Drupal::VERSION,
  deprecatedVersion: '10.3',
  currentCallable: fn() => $renderer->renderInIsolation($elements),
  deprecatedCallable: fn() => $renderer->renderPlain($elements),
);```

Basically now we have to require Drupal 10.3 as our only accepted core version. 
Again that is fine, but in the future we might want to becompatible with multiple versions of core and the above is the way to do it.